src/components/ListItem/ListItem.spec.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/jsx-filename-extension */
+import snap from 'jest-auto-snapshots';
+import 'jest-styled-components';
+
+import ListItem from './ListItem';
```

Comment:
Similar thought to the `Header`, is `ListItem` too broad for a component that's pretty tightly scoped to the Sidebar?